src/endpoints/endpoints.ts

```diff
@@ -0,0 +1,3 @@
+export const Login = {
+  login: "/users/login",
+} as const;
```

Comment:
Let's rename the file to Login, if we want to go with a single file and have constants for each module `endpoints` can be a file instead of a module.